interface/orders/types_ajax.php

```diff
@@ -5,15 +5,15 @@
 // modify it under the terms of the GNU General Public License
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
-
 require_once("../globals.php");
 
+use OpenEMR\Core\Header;
```

Comment:
Is the above class being used in this script?

Reply:
Yes